EpiAware/src/EpiLatentModels/broadcast/latentmodel.jl

```diff
@@ -0,0 +1,61 @@
+@doc raw"
+The `BroadcastLatentModel` struct represents a latent model that supports broadcasting of latent periods.
+
+## Constructors
+- `BroadcastLatentModel(model::M; period::Int, broadcast_rule::B)`: Constructs a `BroadcastLatentModel` with the given `model`, `period`, and `broadcast_rule`.
```

Comment:
These constructors look the same?

Reply:
The first has keywords and the latter doesn't? That does make them different, right?

Reply:
Yeah right made a suggestion